tests/test_deprecation_utils.py

```diff
@@ -13,9 +13,14 @@
 from flask_rebar.utils.deprecation import config as deprecation_config
 
 
-@deprecated_parameters(old_param1="new_param1", old_param2=("new_param2", "v99"))
-def _add(new_param1=42, new_param2=99):
-    return new_param1 + new_param2
+@deprecated_parameters(
+    old_param1="new_param1",
+    old_param2=("new_param2", "v99"),
+    old_param3=("new_param3",),
```

Comment:
Lol, I feel like anyone who passes this is just evil and deserves for their code to bomb.